server/core/region.go

```diff
@@ -617,6 +617,10 @@ func GenerateRegionGuideFunc(enableLog bool) RegionGuideFunc {
 			if len(region.GetPeers()) != len(origin.GetPeers()) {
 				saveKV, saveCache = true, true
 			}
+			if len(region.GetBuckets().GetKeys()) != len(origin.GetBuckets().GetKeys()) {
+				debug("bucket key changed", zap.Uint64("region-id", region.GetID()))
```

Comment:
need to log the key?

Reply:
no need, the only update way  is that the region doesn't inherit the origin region.